addon/models/osf-model.js

```diff
@@ -17,6 +17,7 @@ import HasManyQuery from 'ember-data-has-many-query';
 export default DS.Model.extend(HasManyQuery.ModelMixin, {
     links: DS.attr('links'),
     embeds: DS.attr('embed'),
+    errors: DS.attr(),
```

Comment:
These are embedded parents and roots.  nodes/node_id/?embed=parent.   The embedded parent has no indication what the problem was, just content=null.  It does make a request afterwards to fetch the parent itself which 403's.   I wanted to know if there was any way in the embedded context to preserve this information.  

@hmoco is going to take over looking into this.  Thanks for your helpful comments.